### PR TITLE
feat: footer

### LINKED ISSUE #36: Footer

### PR DESCRIPTION
**Pull request by vonnue-akhil**
image
image
image

Dependency:- 
- EventTitle component
- Button component

closes #36

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { EventVariation } from '../../data';
+import Facebook from '../../assets/footer/facebook.svg?react'
+import FooterData from '../../dummydata/Footer';
+import Instagram from '../../assets/footer/instagram.svg?react'
+import LinkedIn from '../../assets/footer/linkedIn.svg?react'
+import Logo from '../../assets/header/logo.svg?react';
+import Podcast from '../../assets/footer/podcast.svg?react'
+import Twitter from '../../assets/footer/twitter.svg?react'
+import YouTube from '../../assets/footer/youtube.svg?react'
+import EventTitle from '../eventTitle/EventTitle';
```
Comment (Macdeal) [inline src/components/Footer/Footer.tsx:10]: sort

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+						<YouTube className='w-6 h-6' />
+					</a>
+					<a className='p-2 lg:p-4 hover:text-gray-200 transition-all' href="/" >
+						<Podcast className='w-6 h-6' />
+					</a>
+				</div>
+				<span className='uppercase tracking-wider'>{FooterData.copyrightText}</span>
+			</div>
+		</footer>
+	</>
```
Comment (Macdeal) [inline src/components/Footer/Footer.tsx:51]: use default indentation

File: src/index.css [hunk 3]
```diff
@@ -13,6 +13,8 @@
 body {
 	font-family: inter, sans-serif;
 	line-height: 1.5;
+	background-color: rgb(17 24 39);
+	color: #9ca3af;
```
Comment (Macdeal) [inline src/index.css:17]: It should be added to Tailwind

File: src/pages/HomePage.tsx [hunk 4]
```diff
@@ -12,6 +13,7 @@ const HomePage = () => (
       link={tempEventCardData.link}
       title={tempEventCardData.title}
     />
+    <Footer />
```
Comment (Macdeal) [inline src/pages/HomePage.tsx:16]: IMO we need to create a layout  with header and footer, so we can reuse that

**Review by Macdeal — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+					<a className='p-2 lg:p-4 hover:text-gray-200 transition-all' href="/" >
+						<LinkedIn className='w-6 h-6' />
+					</a>
+					<a className='p-2 lg:p-4 hover:text-gray-200 transition-all' href="/" >
+						<YouTube className='w-6 h-6' />
+					</a>
+					<a className='p-2 lg:p-4 hover:text-gray-200 transition-all' href="/" >
+						<Podcast className='w-6 h-6' />
+					</a>
+				</div>
```
Comment (Macdeal) [inline src/components/Footer/Footer.tsx:47]: can we use loop here?

File: src/index.css (hunk 3 above)
Comment (APJ-9) [reply src/index.css:17]: It is common for all the field

File: src/pages/HomePage.tsx (hunk 4 above)
Comment (APJ-9) [reply src/pages/HomePage.tsx:16]: Creating as an issue

File: src/components/Footer/Footer.tsx (hunk 5 above)
Comment (APJ-9) [reply src/components/Footer/Footer.tsx:47]: How is it possible

File: src/components/Footer/Footer.tsx (hunk 2 above)
Comment (APJ-9) [reply src/components/Footer/Footer.tsx:51]: This is my default indentation

File: src/components/Footer/Footer.tsx (hunk 5 above)
Comment (Macdeal) [reply src/components/Footer/Footer.tsx:47]: for a tag

Comment (APJ-9) [reply src/components/Footer/Footer.tsx]: Done

**Pull request merged**